pkg/model/dev.go

```diff
@@ -983,7 +983,19 @@ func (dev *Dev) ToTranslationRule(main *Dev, reset bool) *TranslationRule {
 	}
 
 	if main.PersistentVolumeEnabled() {
+		wd, err := os.Getwd()
+		if err != nil {
+			oktetoLog.Info("could not retrieve working directory")
+		}
 		for _, v := range dev.Volumes {
+			subpath := v.RemotePath
+			if filepath.IsAbs(subpath) {
+				subpath, err = filepath.Rel(wd, v.RemotePath)
+				if err != nil || strings.HasPrefix(subpath, "..") {
+					oktetoLog.Info("could not retrieve subpath")
+					subpath = filepath.Base(v.RemotePath)
+				}
+			}
```

Comment:
I think this logic is the same in both cases, the only thing that change is the remove vs local path. Can we extract the code to a function?